examples/federated_statistics/df_stats/custom/df_stats_generator.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Optional, List
+
+import numpy as np
+import pandas as pd
+from pandas.core.series import Series
+
+from load_data_utils import get_app_paths, load_config
+from nvflare.apis.fl_constant import ReservedKey
+from nvflare.apis.fl_context import FLContext
+from nvflare.app_common.abstract.statistics_spec import Statistics
+from nvflare.app_common.statistics.numeric_stats import (
+    get_std_histogram_buckets,
+    dtype_to_data_type
+)
+from nvflare.app_common.statistics.stats_def import BinRange, Histogram, HistogramType, Feature
+
+
+class DFStatistics(Statistics):
```

Comment:
If we call "Statistics" -> DFStatistics, suggest change file name to df_statistics.py
If we change to "StatisticsGenerator" then suggest change class name to "DFStatisticsGenerator"